Dockerfile

```
@@ -1,11 +1,21 @@
-FROM golang:1.16-alpine
-WORKDIR /go/src/app
-ADD . .
+FROM golang:1.15-alpine AS builder
```

Should we keep 1.16?
```suggestion
FROM golang:1.16-alpine AS builder
```

---

yep, my fault